<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgLogin</class>
 <widget class="QDialog" name="DlgLogin">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>309</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>change password</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLineEdit" name="txtSmAddress">
   <property name="geometry">
    <rect>
     <x>141</x>
     <y>83</y>
     <width>228</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>服务器地址</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 2px solid gray;
    border-radius: 5px;
    padding: 0 4px; 
	
	image: url(:/images/lineedit.png);
	selection-background-color: darkgray;
 }
</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string extracomment="请"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtUserName">
   <property name="geometry">
    <rect>
     <x>141</x>
     <y>120</y>
     <width>228</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>用户名</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 2px solid gray;
    border-radius: 5px;
    padding: 0 4px; 
	
	image: url(:/images/lineedit.png);
	selection-background-color: darkgray;
 }
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPwd">
   <property name="geometry">
    <rect>
     <x>141</x>
     <y>156</y>
     <width>228</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>密码</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 2px solid gray;
    border-radius: 5px;
    padding: 0 4px; 
	
	image: url(:/images/lineedit.png);
	selection-background-color: darkgray;
 }
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLogin">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>230</y>
     <width>84</width>
     <height>41</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton 
	{
		border-image:url(:/images/login_normal.png); 		} 
QPushButton:hover:pressed
	{
		border-image:url(:/images/login_hover.png);
	} 
  QPushButton:hover:!pressed 
	{
		border-image:url(:/images/login_hover.png);
	}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>400</width>
     <height>40</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
   <property name="autoRepeatDelay">
    <number>300</number>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>230</y>
     <width>84</width>
     <height>41</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton 
	{
		border-image:url(:/images/cancel_normal.png); 		} 
QPushButton:hover:pressed
	{
		border-image:url(:/images/cancel_hover.png);
	} 
  QPushButton:hover:!pressed 
	{
		border-image:url(:/images/cancel_hover.png);
	}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>40</height>
    </size>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkRember">
   <property name="geometry">
    <rect>
     <x>149</x>
     <y>188</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox {
     spacing: 5px;
	 color: white ;
 }

 QCheckBox::indicator {
     width: 13px;
     height: 13px;
 }

 QCheckBox::indicator:unchecked {
     image: url(:/images/check_unselected.png);
 }
QCheckBox::indicator:unchecked:hover {
     image: url(:/images/check_skipped_unselected.png);
 }
QCheckBox::indicator:checked {
     image: url(:/images/check_selected.png);
 }
QCheckBox::indicator:checked:hover {
     image: url(:/images/check_skipped_selected.png);
 }</string>
   </property>
   <property name="text">
    <string>记住设置</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkAuto">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>188</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox {
     spacing: 5px;
	color:white;
 }

 QCheckBox::indicator {
     width: 13px;
     height: 13px;
 }

 QCheckBox::indicator:unchecked {
     image: url(:/images/check_unselected.png);
 }
QCheckBox::indicator:unchecked:hover {
     image: url(:/images/check_skipped_unselected.png);
 }
QCheckBox::indicator:checked {
     image: url(:/images/check_selected.png);
 }
QCheckBox::indicator:checked:hover {
     image: url(:/images/check_skipped_selected.png);
 }</string>
   </property>
   <property name="text">
    <string>自动登录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hideButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>最小化</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton 
	{
		border-image:url(:/images/minize_normal.png); 		} 
QPushButton:hover:pressed
	{
		border-image:url(:/images/minize_hover.png);
	} 
  QPushButton:hover:!pressed 
	{
		border-image:url(:/images/minize_hover.png);
	}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="quitButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>关闭</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton 
	{
		border-image:url(:/images/shutdown_normal.png); 		} 
QPushButton:hover:pressed
	{
		border-image:url(:/images/shutdown_hover.png);
	} 
  QPushButton:hover:!pressed 
	{
		border-image:url(:/images/shutdown_hover.png);
	}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnLogin</sender>
   <signal>clicked()</signal>
   <receiver>DlgLogin</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>quitButton</sender>
   <signal>clicked()</signal>
   <receiver>DlgLogin</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>439</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
